### PR TITLE
fix(SplitButton): Remove borders from hover and pressed state in primary split buttons

### LINKED ISSUE #25053: [Bug]: Primary button's width changes when hovered/active
### Library

React / v8 (@fluentui/react)

### System Info

```shell
System:
    OS: Linux 5.10 Ubuntu 20.04 LTS (Focal Fossa)
    CPU: (20) x64 Intel(R) Core(TM) i9-10900K CPU @ 3.70GHz
    Memory: 30.53 GB / 62.67 GB
    Container: Yes
    Shell: 5.8 - /usr/bin/zsh
  Browsers:
    Firefox: 104.0
```

### Are you reporting Accessibility issue?

_No response_

### Reproduction

v8's codepen

## Actual Behavior
Default state:
image

Active/hovered:
image

## Expected Behavior
Primary button should not change width when `active`/`hovered`.

### Logs

_No response_

### Requested priority

Low

### Products/sites affected

_No response_

### Are you willing to submit a PR to fix?

yes

### Validations

- [X] Check that there isn't already an issue that reports the same bug to avoid creating a duplicate.
- [X] The provided reproduction is a minimal reproducible example of the bug.

### PR DESCRIPTION
**Pull request by sopranopillow**
## Current Behavior

When the split button is hovered or pressed, a 1px border is added changing the width based on the rest state.

> See images in issue for more details.

## New Behavior

Split Button no longer adds a border when hovered or pressed.

rest:
image

hovered:
image

pressed:
image

## Related Issue(s)

Fixes #25053

### CODE REVIEW

**Comment by fabricteam** [comment]
## 📊 Bundle size report

<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/e059a9f04ce12cd84892fe6c2f17ef7071ccd898'>e059a9f04ce12cd84892fe6c2f17ef7071ccd898</a></sub>
<div id="bundleSizeComment9423"/>

File: packages/react/src/components/Button/ButtonThemes.ts [hunk 1]
```diff
@@ -157,7 +157,6 @@ export function primaryStyles(theme: ITheme): IButtonStyles {
   return {
     root: {
       backgroundColor: s.primaryButtonBackground,
-      border: `1px solid ${s.primaryButtonBackground}`,
```
Comment (khmakoto) [inline packages/react/src/components/Button/ButtonThemes.ts:160]: Is this not removing the borders from all primary buttons and not only from `SplitButtons`?

**Comment by fabricteam** [comment]
## [Perf Analysis (`@fluentui/react`)](https://github.com/microsoft/fluentui/wiki/Perf-Testing)

  <table>
  <tr>
    <th>Scenario</th>
    <th>Render type</th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">Master Ticks</a>
    </th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">PR Ticks</a>
    </th>
    <th>Iterations</th>
    <th>Status</th>
  </tr><tr>
            <td>TeachingBubble</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25059/perf-test/TeachingBubble-mount_base.html">109382</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25059/perf-test/TeachingBubble-mount.html">110981</a></td>
            <td>5000</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25059/perf-test/TeachingBubble-mount.regression.txt">Possible regression</a></td>
           </tr></table><details><summary>All results</summary><p>
  <table>
  <tr>
    <th>Scenario</th>
    <th>Render type</th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">Master Ticks</a>
    </th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">PR Ticks</a>
    </th>
    <th>Iterations</th>
    <th>Status</th>
  </tr><tr>
            <td>BaseButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25059/perf-test/BaseButton-mount_base.html">1550</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25059/perf-test/BaseButton-mount.html">1527</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Breadcrumb</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25059/perf-test/Breadcrumb-mount_base.html">3542</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25059/perf-test/Breadcrumb-mount.html">3564</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Checkbox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25059/perf-test/Checkbox-mount_base.html">3310</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25059/perf-test/Checkbox-mount.html">3308</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>CheckboxBase</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25059/perf-test/CheckboxBase-mount_base.html">2971</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25059/perf-test/CheckboxBase-mount.html">2990</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ChoiceGroup</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25059/perf-test/ChoiceGroup-mount_base.html">5763</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25059/perf-test/ChoiceGroup-mount.html">5680</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ComboBox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25059/perf-test/ComboBox-mount_base.html">1516</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25059/perf-test/ComboBox-mount.html">1557</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>CommandBar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25059/perf-test/CommandBar-mount_base.html">11690</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25059/perf-test/CommandBar-mount.html">11729</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>ContextualMenu</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25059/perf-test/ContextualMenu-mount_base.html">14054</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25059/perf-test/ContextualMenu-mount.html">14149</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>DefaultButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25059/perf-test/DefaultButton-mount_base.html">1760</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25059/perf-test/DefaultButton-mount.html">1778</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRow</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25059/perf-test/DetailsRow-mount_base.html">4605</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25059/perf-test/DetailsRow-mount.html">4585</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRowFast</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25059/perf-test/DetailsRowFast-mount_base.html">4642</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25059/perf-test/DetailsRowFast-mount.html">4621</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRowNoStyles</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25059/perf-test/DetailsRowNoStyles-mount_base.html">4440</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25059/perf-test/DetailsRowNoStyles-mount.html">4489</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Dialog</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25059/perf-test/Dialog-mount_base.html">3789</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25059/perf-test/Dialog-mount.html">3798</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>DocumentCardTitle</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25059/perf-test/DocumentCardTitle-mount_base.html">702</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25059/perf-test/DocumentCardTitle-mount.html">710</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Dropdown</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25059/perf-test/Dropdown-mount_base.html">4411</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25059/perf-test/Dropdown-mount.html">4163</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FocusTrapZone</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25059/perf-test/FocusTrapZone-mount_base.html">2414</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25059/perf-test/FocusTrapZone-mount.html">2515</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FocusZone</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25059/perf-test/FocusZone-mount_base.html">2392</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25059/perf-test/FocusZone-mount.html">2393</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedList</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25059/perf-test/GroupedList-mount_base.html">67923</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25059/perf-test/GroupedList-mount.html">78425</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedList</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25059/perf-test/GroupedList-virtual-rerender_base.html">32360</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25059/perf-test/GroupedList-virtual-rerender.html">32560</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedList</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25059/perf-test/GroupedList-virtual-rerender-with-unmount_base.html">106942</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25059/perf-test/GroupedList-virtual-rerender-with-unmount.html">107862</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedListV2</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25059/perf-test/GroupedListV2-mount_base.html">707</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25059/perf-test/GroupedListV2-mount.html">669</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedListV2</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25059/perf-test/GroupedListV2-virtual-rerender_base.html">635</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25059/perf-test/GroupedListV2-virtual-rerender.html">659</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedListV2</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25059/perf-test/GroupedListV2-virtual-rerender-with-unmount_base.html">682</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25059/perf-test/GroupedListV2-virtual-rerender-with-unmount.html">666</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>IconButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25059/perf-test/IconButton-mount_base.html">2483</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25059/perf-test/IconButton-mount.html">2475</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Label</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25059/perf-test/Label-mount_base.html">895</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25059/perf-test/Label-mount.html">901</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Layer</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25059/perf-test/Layer-mount_base.html">5264</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25059/perf-test/Layer-mount.html">5278</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Link</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25059/perf-test/Link-mount_base.html">1031</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25059/perf-test/Link-mount.html">1013</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>MenuButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25059/perf-test/MenuButton-mount_base.html">2148</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25059/perf-test/MenuButton-mount.html">2158</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>MessageBar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25059/perf-test/MessageBar-mount_base.html">2775</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25059/perf-test/MessageBar-mount.html">2882</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Nav</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25059/perf-test/Nav-mount_base.html">4117</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25059/perf-test/Nav-mount.html">4144</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>OverflowSet</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25059/perf-test/OverflowSet-mount_base.html">1663</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25059/perf-test/OverflowSet-mount.html">1667</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Panel</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25059/perf-test/Panel-mount_base.html">3152</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25059/perf-test/Panel-mount.html">3132</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Persona</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25059/perf-test/Persona-mount_base.html">1583</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25059/perf-test/Persona-mount.html">1593</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Pivot</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25059/perf-test/Pivot-mount_base.html">2067</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25059/perf-test/Pivot-mount.html">2084</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>PrimaryButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25059/perf-test/PrimaryButton-mount_base.html">1907</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25059/perf-test/PrimaryButton-mount.html">1907</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Rating</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25059/perf-test/Rating-mount_base.html">9134</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25059/perf-test/Rating-mount.html">9155</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SearchBox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25059/perf-test/SearchBox-mount_base.html">1945</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25059/perf-test/SearchBox-mount.html">1995</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Shimmer</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25059/perf-test/Shimmer-mount_base.html">3641</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25059/perf-test/Shimmer-mount.html">3657</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Slider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25059/perf-test/Slider-mount_base.html">2605</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25059/perf-test/Slider-mount.html">2611</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SpinButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25059/perf-test/SpinButton-mount_base.html">5806</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25059/perf-test/SpinButton-mount.html">5864</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Spinner</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25059/perf-test/Spinner-mount_base.html">991</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25059/perf-test/Spinner-mount.html">969</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SplitButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25059/perf-test/SplitButton-mount_base.html">3936</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25059/perf-test/SplitButton-mount.html">3908</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Stack</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25059/perf-test/Stack-mount_base.html">1092</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25059/perf-test/Stack-mount.html">1084</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>StackWithIntrinsicChildren</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25059/perf-test/StackWithIntrinsicChildren-mount_base.html">3119</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25059/perf-test/StackWithIntrinsicChildren-mount.html">3086</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>StackWithTextChildren</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25059/perf-test/StackWithTextChildren-mount_base.html">6634</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25059/perf-test/StackWithTextChildren-mount.html">6352</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SwatchColorPicker</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25059/perf-test/SwatchColorPicker-mount_base.html">12996</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25059/perf-test/SwatchColorPicker-mount.html">13065</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>TagPicker</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25059/perf-test/TagPicker-mount_base.html">3286</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25059/perf-test/TagPicker-mount.html">3356</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>TeachingBubble</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25059/perf-test/TeachingBubble-mount_base.html">109382</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25059/perf-test/TeachingBubble-mount.html">110981</a></td>
            <td>5000</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25059/perf-test/TeachingBubble-mount.regression.txt">Possible regression</a></td>
           </tr>
<tr>
            <td>Text</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25059/perf-test/Text-mount_base.html">985</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25059/perf-test/Text-mount.html">1016</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>TextField</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25059/perf-test/TextField-mount_base.html">2025</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25059/perf-test/TextField-mount.html">2051</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ThemeProvider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25059/perf-test/ThemeProvider-mount_base.html">1854</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25059/perf-test/ThemeProvider-mount.html">1905</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ThemeProvider</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25059/perf-test/ThemeProvider-virtual-rerender_base.html">1322</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25059/perf-test/ThemeProvider-virtual-rerender.html">1334</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ThemeProvider</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25059/perf-test/ThemeProvider-virtual-rerender-with-unmount_base.html">2644</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25059/perf-test/ThemeProvider-virtual-rerender-with-unmount.html">2650</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Toggle</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25059/perf-test/Toggle-mount_base.html">1385</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25059/perf-test/Toggle-mount.html">1390</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>buttonNative</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25059/perf-test/buttonNative-mount_base.html">670</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25059/perf-test/buttonNative-mount.html">659</a></td>
            <td>5000</td>
            <td></td>
           </tr></table></p></details>

<div id="perfComment9423"/>

File: packages/react/src/components/Button/ButtonThemes.ts (hunk 1 above)
Comment (sopranopillow) [reply packages/react/src/components/Button/ButtonThemes.ts:160]: Oh true, I think I have to add a selector instead. Thanks for pointing it out.

File: packages/react/src/components/Button/SplitButton/SplitButton.styles.ts [hunk 2]
```diff
@@ -49,6 +49,12 @@ export const getStyles = memoizeFunction(
                   borderRightWidth: '0',
                   ...getHighContrastNoAdjustStyle(),
                 },
+                ':hover': {
+                  border: '0',
```
Comment (khmakoto) [inline packages/react/src/components/Button/SplitButton/SplitButton.styles.ts:53]: _nit:_ Just a preference of using `none` over `0` but it's your choice.

**Review by khmakoto — APPROVED** [review]

**Pull request merged**